doc/api/buffer.md

```diff
@@ -1058,6 +1058,28 @@ console.log(bufA.length);
 `Buffer.concat()` may also use the internal `Buffer` pool like
 [`Buffer.allocUnsafe()`][] does.
 
+### Static method: `Buffer.copyFrom(view[, offset[, length]])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `view` {TypedArray} The {TypedArray} to copy.
+* `offset` {integer} The starting offset within `view`. **Default:**: `0`.
+* `length` {integer} The number of elements from `view` to copy.
+  **Default:** `view\.length - offset`.
```

Comment:
```suggestion
  **Default:** `view.length - offset`.
```

Reply:
FWIW, the `\` was added by `make format-md`